src/DataViews/FunctionsDataView.cpp

```diff
@@ -36,8 +36,11 @@ FunctionsDataView::FunctionsDataView(AppInterface* app,
     : DataView(DataViewType::kFunctions, app, metrics_uploader) {}
 
 const std::string FunctionsDataView::kUnselectedFunctionString = "";
-const std::string FunctionsDataView::kSelectedFunctionString = "✓";
+const std::string FunctionsDataView::kSelectedFunctionString = "H";
 const std::string FunctionsDataView::kFrameTrackString = "F";
+const std::string FunctionsDataView::kApiScopeType = "MS";
+const std::string FunctionsDataView::kApiScopeAsyncType = "MA";
+const std::string FunctionsDataView::kHookedFunctionType = "D";
```

Comment:
The suffix of these constants should be `String` like the other ones, not `Type` as they don't represent a type. E.g.: `kApiScopeTypeString`.